### PR TITLE
Dasd 8306 migrate cluster

### PR DESCRIPTION
**Pull request by NickGraham101**

### CODE REVIEW

File: pipeline-templates/stage/build.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        displayName: 'Copy Files to: $(build.artifactstagingdirectory)'
+        inputs:
+          contents: |
+            azure/**
+            manifests/**
+          targetFolder: '$(build.artifactstagingdirectory)/publish'
+         
+  - job: 'build_dacpac'
+    pool:
+      vmImage: 'windows-2019'
```
Comment (nbowes24) [inline pipeline-templates/stage/build.yml:49]: Using azure hosted agents here

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,3 +1,17 @@
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.3.0
```
Comment (nbowes24) [inline azure-pipelines.yml:7]: 0.3.1 contains a fix for kubectl apply

File: azure-pipelines.yml [hunk 3]
```diff
@@ -6,87 +20,70 @@ trigger:
...
-      - task: Docker@2
-        displayName: Build Image
-        inputs:
-          command: build
-          dockerfile: Dockerfile
-          repository: apprenticeshipsdevops/das-tools-service
-          tags: |
-            $(build.buildnumber)
+- name: DatabaseConnectionString
+  value: Server=tcp:$(SharedSQLServerFQDN),1433;Database=das-$(ResourceEnvironmentName)-$(ServiceName)-db;User ID=$(SQLServerServiceAccountUsername);Password=$(SQLServerServiceAccountPassword);Trusted_Connection=False;Pooling=False;Connect Timeout=200;MultipleActiveResultSets=True
```
Comment (adam230594) [inline azure-pipelines.yml:32]: Is it worth having this as a variable in a variable group?

File: manifests/deployment.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ metadata:
     app: das-tool-service-web
     service: das-tool-service
   name: das-tool-service-web
-  namespace: {{Namespace}}
+  namespace: {{KubernetesNamespace}}
 spec:
```
Comment (nbowes24) [inline manifests/deployment.yml:10]: Does this now need a nodeselector?

File: pipeline-templates/stage/build.yml (hunk 1 above)
Comment (NickGraham101) [reply pipeline-templates/stage/build.yml]: Changed agent pool

File: azure-pipelines.yml (hunk 2 above)
Comment (NickGraham101) [reply azure-pipelines.yml:7]: Upgraded

File: azure-pipelines.yml (hunk 3 above)
Comment (NickGraham101) [reply azure-pipelines.yml:32]: Moved to variable group

File: manifests/deployment.yml (hunk 4 above)
Comment (NickGraham101) [reply manifests/deployment.yml:10]: Added node selector

**Review by adam230594 — APPROVED** [review]

**Review by nbowes24 — APPROVED** [review]

**Pull request merged**